src/containers/navigation-icons/NavigationIcons.styles.ts

```diff
@@ -14,5 +15,14 @@ export const styles = {
     ...hideOnMobile,
     ml: '12px'
   },
-  studentIcons: { ...hideOnMobile, color: 'primary.900' }
+  studentIcons: {
+    ...hideOnMobile,
+    color: 'primary.900'
+  },
+  accountIcon: {
+    ...hideOnMobile,
+    color: 'primary.900',
```

Comment:
can you also move color property to hideOnMobile?

Reply:
Moved. Thank you